cmake/SelectPcsx2Plugins.cmake

```diff
@@ -5,7 +5,7 @@ set(msg_dep_common_libs "check these libraries -> wxWidgets (>=2.8.10), aio")
 set(msg_dep_pcsx2       "check these libraries -> wxWidgets (>=2.8.10), gtk2 (>=2.16), zlib (>=1.2.4), pcsx2 common libs")
 set(msg_dep_cdvdiso     "check these libraries -> bzip2 (>=1.0.5), gtk2 (>=2.16)")
 set(msg_dep_zerogs      "check these libraries -> glew (>=1.6), opengl, X11, nvidia-cg-toolkit (>=2.1)")
-set(msg_dep_gsdx        "check these libraries -> opengl, png++, X11")
+set(msg_dep_gsdx        "check these libraries -> opengl, png (>=1.2), X11")
```

Comment:
iirc there were some changes in later libpng versions so you can't directly use latest ones with 1.2 one.